backend/danswer/connectors/web/connector.py

```diff
@@ -133,9 +135,20 @@ def load_from_state(self) -> GenerateDocumentsOutput:
                 title = None
                 if title_tag and title_tag.text:
                     title = title_tag.text
+                    title_tag.extract()
 
-                # Heuristics based cleaning
-                for undesired_div in ["sidebar", "header", "footer"]:
+                # Heuristics based cleaning of links
```

Comment:
Since this is specific to Sphinx documentation, can we add a comment mentioning this and maybe linking to sphinx page?

Reply:
Ah I see. In case you start with some index page? Make sense. I will remove it.